packages/dart_frog_auth/README.md

```diff
@@ -149,6 +149,97 @@ Response onRequest(RequestContext context) {
 
 In the case of `null` be returned (unauthenticated), the middleware will automatically send an unauthorized `401` in the response.
```

Comment:
I have a few changes I'd like to make to this README so I'll wait until after this is merged to do so